build/keycloak/docker-entrypoint.sh

```diff
@@ -17,44 +17,10 @@ if [ -z "${EXTERNAL_ADDR}" ]; then
 fi
 export EXTERNAL_ADDR
 
-# Add admin user
-if [ -n "${KEYCLOAK_USER}" ] && [ -n "${KEYCLOAK_PASSWORD}" ]; then
-  /opt/jboss/keycloak/bin/add-user-keycloak.sh --user "${KEYCLOAK_USER}" --password "${KEYCLOAK_PASSWORD}"
-fi
-
-# Default to H2 if DB type not detected
```

Comment:
Could consider leaving some of these checks around, but I don't think it was adding much, given how database is configured in newer keycloak versions.

Reply:
lgtm